lib/mumuki/laboratory/mumukit/randomizations/with_randomizations.rb

```diff
@@ -0,0 +1,36 @@
+module WithRandomizations
+  extend ActiveSupport::Concern
+
+  included do
+    serialize :randomizations, Hash
+
+    include InstanceMethods
```

Comment:
You don't need to extend `ClassMethods`, it is done automatically. 

Also, don't create an `InstanceMethods` mixin. Just embed the instance methods in the mixin. Please have a look at other concerns. 

Finally, put this code directly in the concerns folder, since it is laboratory-specific.